pkg/tsdb/cloudwatch/log_query.go

```diff
@@ -0,0 +1,67 @@
+package cloudwatch
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func logsResultsToDataframes(response *cloudwatchlogs.GetQueryResultsOutput) (*data.Frame, error) {
```

Comment:
Some tests for this function would be nice. I *think* it should be possible to make it less complex but I'm not sure. 

Tests would make it easier to refactor this if it's possible.

Reply:
👍 Sure, will do